data/sidebars/developerToolsSidebar.ts

```diff
@@ -1,79 +1,81 @@
-import { SidebarSection } from "../types";
+import { SidebarContent } from "../types";
 
-export const DEVELOPER_TOOLS_SIDEBAR: SidebarSection[] = [
+const baseSlug = "/developer-tools";
+
+export const parentSection = {
+  slug: baseSlug,
+  title: "Developer tools",
+};
+
+export const DEVELOPER_TOOLS_SIDEBAR_CONTENT: SidebarContent[] = [
```

Comment:
```suggestion
export const DEVELOPER_TOOLS_SIDEBAR: SidebarContent[] = [
```
nit but the other sidebars don't use the `_CONTENT` suffix. You can keep if you want nbd